assets/src/views/AchievedGoals.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <div class="achieved-home">
+    <AchievedList :httpClient="getHttpClient" />
+  </div>
+</template>
+
+<script>
+// @ is an alias to /src
+import AchievedList from '@/components/AchievedList.vue';
```

Comment:
I like the use of @ here as an alias to the src folder... wondering if it also be used in the following import statement `import HEADERS from '@/constants/http';` of `assets/src/services/serverWrapper.js` above?

Reply:
Ooh, thanks for pointing this out. I missed quite a few! Fixed in 74d8baa